homeassistant/components/lg_thinq/strings.json

```diff
@@ -993,6 +993,28 @@
           "normal": "[%key:component::lg_thinq::entity::sensor::current_dish_washing_course::state::delicate%]"
         }
       }
+    },
+    "water_heater": {
+      "water_boiler": {
+        "name": "Hot water",
```

Comment:
in what devices is this used? Isn't the water heater the main entity of these devices? Should we set the name to `None` in the entity so the entity name becomes the device name